q2_feature_table/tests/test_summarize.py

```diff
@@ -18,7 +18,9 @@
 from q2_types.feature_data import DNAIterator
 import csv
 
-from q2_feature_table import tabulate_seqs, summarize
+from q2_feature_table import (
```

Comment:
We should have tests of `summarize_plus` as well.

Reply:
It almost feels redundant testing this since it just calls 3 actions and doesn't use the output from one as input to another it just calls them straight. Do you think we should bother covering all cases for `summarize_plus` or just the base case?

Reply:
Just a simple base case will be fine.